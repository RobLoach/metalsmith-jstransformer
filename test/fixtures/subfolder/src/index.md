---
layout: layouts/pages/blog-post.jade
---

# Hello World!

This is a test.

