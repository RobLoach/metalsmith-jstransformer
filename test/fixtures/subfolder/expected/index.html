<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Jade</title>
  </head>
  <body><div class="blog-post"><h1>Hello World!</h1>
<p>This is a test.</p>
</div></body>
</html>